src/VisualStudio/LiveShare/Impl/CustomProtocol/Projects.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.VisualStudio.LanguageServices.LiveShare.CustomProtocol
+{
+    /// <summary>
+    /// TODO - This custom liveshare model should live elsewhere.
```

Comment:
same here. basically work item for all todo.

Reply:
https://github.com/dotnet/roslyn/projects/45#card-20619634